internal/provider/resource_consul_cluster.go

```diff
@@ -449,6 +458,10 @@ func setConsulClusterResourceData(d *schema.ResourceData, cluster *consulmodels.
 		return err
 	}
 
+	if err := d.Set("auto_hvn_to_hvn_peering", cluster.Config.AutoHvnToHvnPeering); err != nil {
```

Comment:
Does the Consul create operation include the HVN peering creation workflows? So once we receive the operation saying the Consul creation is complete, we can assume all peerings have been created too?

If so, I think a comment would be useful to share that hidden context.

Reply:
Yeah, that's the case. Where would you like this comment to go? Right where you commented? Or in the user facing docs?

Reply:
I put a note right where you commented, because I don't think that's relevant for the user.